src/kube_vxlan_controller_pod_reg.erl

```diff
@@ -117,10 +117,10 @@ terminate(_Reason, _State) ->
 %%% Internal functions
 %%%===================================================================
 
-start_pod(Id, Event, Config, _State) ->
+start_pod(Id, Event, Config, State) ->
     case ?PodSup:start_pod(Id, Event, Config) of
 	{ok, Pid} ->
-	    ets:insert_new(?SERVER, {Id, Pid, undefined});
+            {reply, ok, State, {register, Pid, Id, undefined}};
 	Other ->
 	    ?LOG(debug, "failed to start pod for ~p with ~p", [Id, Other]),
 	    ok
```

Comment:
should have see this earlier, this needs to return a valid regine result.
```suggestion
	    {reply, Other, State}
```

Reply:
I will do it for better/cleaner code undrestanding, but it should be valid because of https://github.com/travelping/regine/blob/master/src/regine_server.erl#L205?

Reply:
fixed now